src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.netstandard.cs

```diff
@@ -311,6 +311,94 @@ private static float MinMaxCore<TMinMaxOperator>(ReadOnlySpan<float> x, TMinMaxO
             return result;
         }
 
+        private static int IndexOfMinMaxCore<TIndexOfMinMaxOperator>(ReadOnlySpan<float> x, TIndexOfMinMaxOperator op = default)
+            where TIndexOfMinMaxOperator : struct, IIndexOfOperator
+        {
+            // This matches the IEEE 754:2019 `maximum`/`minimum` functions.
+            // It propagates NaN inputs back to the caller and
+            // otherwise returns the index of the greater of the inputs.
+            // It treats +0 as greater than -0 as per the specification.
+
+            int result;
+            int i;
+
+            if (Vector.IsHardwareAccelerated && Vector<int>.Count <= 8 && x.Length >= Vector<float>.Count)
+            {
+                ref float xRef = ref MemoryMarshal.GetReference(x);
+
+                Vector<int> resultIndex = new Vector<int>([0, 1, 2, 3, 4, 5, 6, 7], 0);
```

Comment:
Two things:
1. I forgot there's no ReadOnlySpan-based constructor in the netstandard20 surface area of vector. That means this is still going to be allocating a new array on every use.
2. The `,0` is superflous.

Net result is we should cache this array, e.g.:
```C#
private static readonly int[] s_0through7 = [0, 1, 2, 3, 4, 5, 6, 7];
```
and then use it here:
```C#
Vector<int> resultIndex = new Vector<int>(s_0through7);
```

Reply:
Fixed.